<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="6" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-10-17T17:29:10 IST" name="Default suite" finished-at="2021-10-17T17:29:10 IST" duration-ms="241">
    <groups>
    </groups>
    <test started-at="2021-10-17T17:29:10 IST" name="Default test" finished-at="2021-10-17T17:29:10 IST" duration-ms="241">
      <class name="calculator.StringCalculatorShould">
        <test-method is-config="true" signature="init()[pri:0, instance:calculator.StringCalculatorShould@7671cb68]" started-at="2021-10-17T17:29:10 IST" name="init" finished-at="2021-10-17T17:29:10 IST" duration-ms="15" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method signature="emptyStringReturnzero()[pri:0, instance:calculator.StringCalculatorShould@7671cb68]" started-at="2021-10-17T17:29:10 IST" name="emptyStringReturnzero" finished-at="2021-10-17T17:29:10 IST" duration-ms="30" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- emptyStringReturnzero -->
        <test-method signature="negativeNumberException()[pri:0, instance:calculator.StringCalculatorShould@7671cb68]" started-at="2021-10-17T17:29:10 IST" name="negativeNumberException" finished-at="2021-10-17T17:29:10 IST" duration-ms="4" status="PASS">
          <exception class="java.lang.Exception">
            <message>
              <![CDATA[Negative Input ]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Exception: Negative Input 
at calculator.StringCalculator.findDangerousInput(StringCalculator.java:204)
at calculator.StringCalculator.getSum(StringCalculator.java:188)
at calculator.StringCalculator.calculator(StringCalculator.java:181)
at calculator.StringCalculatorShould.negativeNumberException(StringCalculatorShould.java:67)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Exception -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- negativeNumberException -->
        <test-method signature="singleValueIsReplied()[pri:0, instance:calculator.StringCalculatorShould@7671cb68]" started-at="2021-10-17T17:29:10 IST" name="singleValueIsReplied" finished-at="2021-10-17T17:29:10 IST" duration-ms="6" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- singleValueIsReplied -->
        <test-method signature="threeNumbersDelimitedBoth()[pri:0, instance:calculator.StringCalculatorShould@7671cb68]" started-at="2021-10-17T17:29:10 IST" name="threeNumbersDelimitedBoth" finished-at="2021-10-17T17:29:10 IST" duration-ms="8" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- threeNumbersDelimitedBoth -->
        <test-method signature="twoNumberNewLineDilimter()[pri:0, instance:calculator.StringCalculatorShould@7671cb68]" started-at="2021-10-17T17:29:10 IST" name="twoNumberNewLineDilimter" finished-at="2021-10-17T17:29:10 IST" duration-ms="8" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- twoNumberNewLineDilimter -->
        <test-method signature="twoNumbersCommaDelimited()[pri:0, instance:calculator.StringCalculatorShould@7671cb68]" started-at="2021-10-17T17:29:10 IST" name="twoNumbersCommaDelimited" finished-at="2021-10-17T17:29:10 IST" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- twoNumbersCommaDelimited -->
      </class> <!-- calculator.StringCalculatorShould -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
